sql/changes/sc.04.08.0510.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sensor ADD COLUMN interfaceid INTEGER REFERENCES interface;
```

Comment:
May I suggest a `ON DELETE SET NULL ON UPDATE CASCADE` as part of this definition? Otherwise, interface deletion would not be possible until sensors had been removed.

Reply:
You may indeed